engine/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/EmailPayloadAggregatorFactory.java

```diff
@@ -23,14 +24,14 @@ private EmailPayloadAggregatorFactory() {
 
     }
 
-    public static AbstractEmailPayloadAggregator by(EventAggregationCriterion aggregationKey, LocalDateTime start, LocalDateTime end) {
+    public static AbstractEmailPayloadAggregator by(EventAggregationCriterion aggregationKey, String username, Set<SubscribedEventTypeSeverities> userSeverities) {
```

Comment:
As we already discussed, aggregating by user can lead to a gigantic resources consumption if a lot of people subscribe to the daily digest. I'm worried about this. The whole aggregation logic needs to be reviewed and improved early next year. Please create a Jira ticket about it.